sources/packages/web/src/types/contracts/DataTableContract.ts

```diff
@@ -437,3 +437,15 @@ export const CASInvoicesBatchesHeaders = [
     key: "actions",
   },
 ];
+
+/**
+ * Student application summary headers.
+ */
+export const StudentApplicationsSummaryHeaders = [
+  { title: "Application Number", sortable: true, key: "applicationNumber" },
```

Comment:
Minor. In the existing view the header name is `Application #`. I honestly like `Application Number` better, but letting you know the existing table headers and good to share (fyi) to business and change it.
![image](https://github.com/user-attachments/assets/10949055-95ac-4b7d-8345-69fc67a5e5cf)

Reply:
"Application #" was the old direction and we have been asked to change it to "Application Number".

Reply:
I am good as the business is on same page. thanks.